spec/omniauth/auth0/jwt_validator_spec.rb

```diff
@@ -133,16 +133,26 @@
     end
 
     context 'different from domain' do
-      let(:jwt_validator) do
-        make_jwt_validator(opt_issuer: 'different.auth0.com')
+      shared_examples_for 'has correct issuer and domain' do
+        let(:jwt_validator) { make_jwt_validator(opt_issuer: opt_issuer) }
+
+        it 'should have the correct issuer' do
```

Comment:
do not use `should` just name the example as `it 'has the correct issuer' do`

Reply:
although i agree with you, the use of `should` follows existing pattern, and i figured it's probably worth staying consistent.

Reply:
oh my bad not familiar with set pattern in project :)